src/components/pagination-controller/pagination-controller.js

```diff
@@ -61,18 +60,14 @@ const PaginationController = () => {
       </IconButton>
       <IconButton
         onClick={handleNextButtonClick}
-        disabled={Boolean(
-          Math.ceil(currentPage >= itemsCount / rowsPerPage - 1)
-        )}
+        disabled={Boolean(currentPage >= itemsCount / rowsPerPage - 1)}
```

Comment:
You don't need to use `Boolean`, because `>=` operator returns boolean value.

Reply:
fixed